bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/IteratorValue.java

```diff
@@ -48,12 +50,17 @@ default String stringValue() {
     }
 
     @Override
-    public default Object copy(Map<Object, Object> refs) {
+    default BString bStringValue() {
+        return StringUtils.fromString(stringValue());
+    }
+
+    @Override
+    default Object copy(Map<Object, Object> refs) {
         throw new UnsupportedOperationException();
     }
 
     @Override
-    public default Object frozenCopy(Map<Object, Object> refs) {
+    default Object frozenCopy(Map<Object, Object> refs) {
```

Comment:
don't use explicit default keyword. is there is a reason for this?

Reply:
It was already there. I just removed the public keyword.